config/locales/server.en.yml

```diff
@@ -2584,6 +2585,7 @@ en:
       create_tag_allowed_groups: "min_trust_to_create_tag"
       send_email_messages_allowed_groups: "min_trust_to_send_email_messages"
       skip_review_media_groups: "review_media_unless_trust_level"
+      min_trust_to_post_embedded_media: "embedded_media_allowed_groups"
```

Comment:
I think this is inverted. (I would know because I got this backwards twice already. 😛)